app/screens/post_options/__snapshots__/post_options.test.js.snap

```diff
@@ -10,8 +10,8 @@ exports[`PostOptions should match snapshot, no option for system message to user
 >
   <Connect(SlideUpPanel)
     allowStayMiddle={false}
-    initialPosition={322}
-    marginFromTop={278}
+    initialPosition={372}
+    marginFromTop={228}
```

Comment:
@enahum not sure what is causing this test to change this, so not sure whether there is something wrong with my code, or the snapshot is brittle.